pkg/apk/implementation.go

```diff
@@ -521,11 +523,6 @@ func (a *APK) fetchAlpineKeys(ctx context.Context, versions []string) error {
 	if err := json.Unmarshal(b, &releases); err != nil {
 		return fmt.Errorf("failed to unmarshal alpine releases: %w", err)
 	}
-	// what is the alpine version we use?
-	var alpineVersions = versions
-	if len(alpineVersions) == 0 {
```

Comment:
This is a breaking change, for users who had only specified packages and no repos, which would previously have defaulted to alpine's latest stable release and now would not fetch alpine keys and fail.

I don't know of anybody doing that today, and if someone does do it the solution is to be explicit and specify the alpine repo you want to pull from.